cl/search/tasks.py

```diff
@@ -781,6 +788,15 @@ def update_children_docs_by_query(
     ) as exc:
         handle_ubq_retries(self, exc, count_query=count_query)
 
+    except ApiError as exc:
+        # Catch any ApiError exceptions to handle specific error message.
+        # If the error is search_context_missing_exception retry it.
+        if (
+            exc.info.get("error", {}).get("type", {})
+            == "search_context_missing_exception"
+        ):
```

Comment:
Wouldn't it make more sense to put this in the `handle_ubq_retries` function?

Reply:
Yes, this makes more sense. So, we can continue to add parsing for additional exceptions that should be retried for UBQ requests as we detect them. Changed.